src/corehost/cli/deps_resolver.cpp

```diff
@@ -303,25 +313,40 @@ bool deps_resolver_t::probe_deps_entry(const deps_entry_t& entry, const pal::str
     return false;
 }
 
-bool report_missing_assembly_in_manifest(const deps_entry_t& entry)
+bool report_missing_assembly_in_manifest(const deps_entry_t& entry, bool isWarning)
 {
-    trace::error(_X(
-        "Error:\n"
-        "  An assembly specified in the application dependencies manifest (%s) was not found:\n"
-        "    package: '%s', version: '%s'\n"
-        "    path: '%s'"),
-        entry.deps_file.c_str(), entry.library_name.c_str(), entry.library_version.c_str(), entry.relative_path.c_str());
+    if (!isWarning && entry.asset_type == deps_entry_t::asset_types::resources)
+    {
+        // Treat missing resource assemblies as a warning, not an error.
+        isWarning = true;
+    }
+
+    if (isWarning)
+    {
+        trace::warning(MissingAssemblyMessage.c_str(), _X("Warning"),
+            entry.deps_file.c_str(), entry.library_name.c_str(), entry.library_version.c_str(), entry.relative_path.c_str());
+    }
```

Comment:
We don't want a warning either. We are going to be relying on this behavior in production when the language pack is not installed.